openai.audio/client.bal

```diff
@@ -60,6 +60,7 @@ public isolated client class Client {
     # Transcribes audio into the input language.
     #
     # + return - OK 
+    @display {label: "Create Transcription"}
```

Comment:
Shouldn't we modify the client level annotation as well, to fix the connector name shown in choreo marketplace?

Reply:
I actually did those too but haven't pushed them. Please check now.